index.html

```diff
@@ -11512,7 +11512,7 @@ <h2>Acknowledgements</h2>
 <!-- Simon Stewart --> <a href=http://www.rocketpoweredjetpants.com/>Simon Stewart</a>,
 <!-- Sri Harsha --> Sri Harsha,
 <!-- Titus Fortner --> Titus Fortner,
-<!-- Vangelis Katsikaros --> and Vangelis Katsikaros.
+<!-- Vangelis Katsikaros --> and <a href=http://www.vkatsikaros.com/>Vangelis Katsikaros</a>.
```

Comment:
While quickly discussing this addition we tend more to remove the URLs all together given that it we cannot guarantee that the page is still owned by that person. Instead we could add the GitHub handle.

```suggestion
<!-- Vangelis Katsikaros --> and <a href=https://github.com/vkatsikaros>Vangelis Katsikaros</a>.
```

But we should most likely do this on a different PR for everyone.

Reply:
@whimboo makes sense, we expect the spec to be around for a long time (decades even?). Sorry for the noise!